tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -2384,6 +2384,53 @@ public function test_remove_insecure_properties_applies_safe_styles() {
 		$this->assertEqualSetsWithIndex( $expected, $actual );
 	}
 
+	/**
+	 * @ticket 57321
+	 */
+	public function test_allow_indirect_properties() {
```

Comment:
```suggestion
	public function test_remove_insecure_properties_should_allow_indirect_properties() {
```
This makes the test name more descriptive when called with `phpunit --group 57321 --testdox`.